README.md

```diff
@@ -5,6 +5,68 @@ https://flutter.dev/
 Flutter in an open Source SDK for creating high-performance mobile apps for IOS and Android.
 The Flutter makes it easier for you to build user interfaces, while reducing the amount of code required to create and update your app.
 
+## Why use Flutter?
+
+- Flutter can be used to build applications for Android and IOS
+- The Dart language used in Flutter is quite similar to Java
+- We can update the application in real time
+- If we close the application when we open it again we can continue from where we were
+- Flutter benefits from several objects that help us to facilitate the construction process
```

Comment:
```suggestion
- Flutter has a _complete_ design system with a library of Material UI widgets included which speed up the development process.
```
what is an "object" in the context of an object oriented language? 
and how _many_ is _several_? Is it 10? or 1000?
_all_ app development platforms have "several objects" the question on people's mind is: 
"does it have _everything_ I need to build my app?"
To which the answer is: Yes!